src/app/modules/main/stickers/models/sticker_pack_list.nim

```diff
@@ -123,6 +124,17 @@ QtObject:
     let packInfo = self.packs[self.packIdToRetrieve]
     result = newQVariant(packInfo.stickers)
 
+  # We cannot return QVariant from the proc which has arguments.
+  # First findStickersById has to be called, then getFoundStickers
+  proc findStickersById*(self: StickerPackList, packId: string) {.slot.} =
+    self.foundStickers = newQVariant()
+    for item in self.packs:
+      if(item.pack.id == packId):
+        self.foundStickers = newQVariant(item.stickers)
```

Comment:
Nice to have loop break here after pack.id is matching

Reply:
done